spec.json

```diff
@@ -362,7 +359,40 @@
             ],
             "inheritance": "motor"
         },
-        "dcMotor": {
+        "nxtMotor": {
```

Comment:
I think we should do something like we did for `specialSensorTypes` for these, because they should be able to be completely automatically generated.